geode-core/src/main/java/org/apache/geode/internal/process/MBeanProcessController.java

```diff
@@ -228,9 +230,9 @@ private void disconnect() {
    * @throws PidUnavailableException if parsing the pid from the RuntimeMXBean name fails
    */
   boolean checkPidMatches() throws IllegalStateException, IOException, PidUnavailableException {
```

Comment:
This appears to be unused?  The JavaDoc has it as "NOT USED EXCEPT IN TEST."  This is perhaps no longer true?